### ISSUE TITLE
Add a built in Import Tool

### ISSUE THREAD

**Issue by HanzoX7** [Microsoft/sqlopsstudio#200]
- SQL Operations Studio Version: 0.23.6

Steps to Reproduce:

1. Try to load data from a CSV or Excel file to a table.
2. Notice the lack of an option to do so.

It would be very good if there was some built-in tool that could import external files to into a table... a trimmed down version of the SQL Server Import and Export Wizard that supports CSV or Excel files would be nice.

**Comment by erickangMSFT** [comment]
@HanzoX7 if this feature is available as a CLI, would it satisfy the need?

**Comment by HanzoX7** [comment]
Personally, I'd prefer a graphical tool with an option to output a script to run in CLI, but a CLI tool would be a great start.

**Comment by yualan** [comment]
Have you tried the SQL Server Import extension? https://docs.microsoft.com/en-us/sql/azure-data-studio/sql-server-import-extension?view=sql-server-2017

**Comment by HanzoX7** [comment]
Yes... I've tried it. It's a step in the right direction, but it's kinda limited. It needs to support Excel files and append rows to a table, rather than creating a new table for every imported file.

**Comment by kburtram** [comment]
I've updated the title to reflect the remaining suggestion related to this issue.  Please log additional issues for any gaps with the SQL Server Import extension.  Thanks!

**Comment by yualan** [comment]
Thanks Karl, we are currently investigating this feature request, may be able to incorporate into our Import extension as another supported file type.

**Comment by vtruong68** [comment]
This is a must. I have to interrupt my workflow and convert to .csv or use SSMS to import data.

**Comment by SQLvariant** [comment]
How about just firing up a PowerShell terminal and using the ImportExcel [PowerShell module](https://www.powershellgallery.com/packages/ImportExcel)?  I use this approach all the time and it works ~great~ phenomenally. 

Demo script available right here: https://github.com/SQLvariant/Demos/blob/master/Presentations/Ignite2018-THR2114/2A_Import-Excel.ps1
There's also a video of me demonstrating that script if you navigate up in that repository.

**Comment by HanzoX7** [comment]
@SQLvariant that looks promising... will certainly take a look.

**Comment by dirq** [comment]
This is a must have.  It's been years without a decent Excel import in either SSMS or Data Studio.
